idb/postgres/internal/writer/writer.go

```diff
@@ -11,196 +11,123 @@ import (
 	"github.com/algorand/go-algorand/data/transactions"
 	"github.com/algorand/go-algorand/ledger/ledgercore"
 	"github.com/algorand/go-algorand/protocol"
+	"github.com/jackc/pgx/v4"
```

Comment:
different groups for algorand / third party packages

Reply:
We already have code where those are together. I think we should leave it for a separate discussion. My take is that if we had a linter that warned about the imports, it would not be so smart to allow these two separate.

Reply:
Right, I pointed out the missing groups in one of your earlier PRs but you didn't update them so things are becoming more inconsistent...

Reply:
Or rather more consistent :)

Reply:
+1 for a linter - https://github.com/algorand/go-algorand-internal/issues/1431